### ISSUE TITLE
Cannot switch between browse and focus modes in Excel when UIA is enabled.

### ISSUE THREAD

**Issue by msviolafangirl** [nvaccess/nvda#12881]
### Steps to reproduce:
1. Go into advanced settings and tick"Use UI Automation to access Excel spreadsheet controls when available"
2. 2. Open an excel spreadsheet.
3. 3. Press NVDA+Space to enter browse mode.

### Actual behavior:
When I press NVA+Space to switch between browse and focus modes, I get absolutely no response. This occurs with UIA support enabled in advanced settings but when it turns off, I can cycle between browse and focus modes as intended.

### Expected behavior:
I should be able to press NVDA+Space and switch between browse and focus modes in Excel with UIA turned on.

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
2021.2

#### Windows version:
Windows 10 21H1 x64

#### Name and version of other software in use when reproducing the issue:
Exel 16/365

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
yes

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
